tools/src/main/java/org/apache/rocketmq/tools/command/topic/TopicListSubCommand.java

```diff
@@ -65,7 +65,7 @@ public void execute(final CommandLine commandLine, final Options options,
         try {
             defaultMQAdminExt.start();
             if (commandLine.hasOption('c')) {
-                ClusterInfo clusterInfo = defaultMQAdminExt.examineBrokerClusterInfo();
+                ClusterInfo clusterInfo = defaultMQAdminExt.examineBrokerClusterInfo(null);
```

Comment:
null as parameter is not a good practice

Reply:
Thanks,I will change it in the next commit.